src/common/hashing/password-encoder.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import * as bcrypt from 'bcrypt';
+
+@Injectable()
+export class PasswordEncoderService {
+  async hash(password: string): Promise<string> {
+    const saltOrRounds = 10;
```

Comment:
1. 상수는 UPPER_SNAKE_CASE로 선언하는게 일반적입니다
2. SALT_OR_ROUNDS 를 hash의 지역변수로 둔다면 메서드 실행시마다 변수를 할당하게 되는데 클래스 멤버변수로 두는건 어떨까요?? (성능상 큰 차이는 업음)
   1. hash 메서드에서만 사용하게끔 감추시려는 의도였다면 이부분은 무시하셔도 됩니다

```suggestion
export class PasswordEncoderService {
  private SALT_OR_ROUNDS = 10;

  async hash(password: string): Promise<string> {
```